.github/workflows/integration-tests-matrix.json

```diff
@@ -14,5 +14,9 @@
   {
     "name": "bridge-hub-polkadot",
     "package": "bridge-hub-polkadot-integration-tests"
+  },
```

Comment:
also this needs to be added to the `runtimes-matrix.json` - for running tests/check-migrations...

Reply:
fixed here: https://github.com/polkadot-fellows/runtimes/pull/245